src/HttpRequest.h

```diff
@@ -285,5 +285,10 @@ void UpdateRequestNotes(ConnStateData *csd, HttpRequest &request, NotePairs cons
 /// nil parameter(s) indicate missing caller information and are handled safely
 const Ip::Address *FindListeningPortAddress(const HttpRequest *, const AccessLogEntry *);
 
+/// \returns listening/*_port port number used by the client connection (or zero)
+/// nil parameter(s) indicate missing caller information and are handled safely
+unsigned short FindListeningPortNumber(const HttpRequest *, const AccessLogEntry *);
```

Comment:
We can rename the old/existing FindListeningPortAddress() function to FindListeningPortIp() to improve the distinction between the two similar functions. I did not rename, keeping the diff small and to-the-point, but I am happy to rename if others think it is a good idea.

Reply:
Ah, the multiple meanings of "port".

"PortIp" looks like it is missing the word "And" which would be wrong. "Port Address" is correct for the detail being located.

Reply:
I am OK with leaving FindListeningPortAddress() name as is, but if you prefer FindListeningPortIpAddress() or some other name, please let me know.